test.js

```diff
@@ -1,29 +1,19 @@
-'use strict';
-var test = require('ava');
-var nircmd = require('./');
+import test from 'ava';
+import fn from './';
 
-test('clipboard', function (t) {
-	t.plan(1);
-
-	nircmd('clipboard set "hey everybody"', function (err) {
-		t.assert(!err, err);
-	});
+test('clipboard', async t => {
+	await nircmd('clipboard set "hey everybody"');
 });
 
-test('screensaver', function (t) {
-	t.plan(1);
-
-	nircmd('screensaver', function (err) {
-		t.assert(!err, err);
-	});
+test('screensaver', async t => {
+	await nircmd('screensaver');
 });
 
-test('screensaver using spawn', function (t) {
-	t.plan(1);
-
-	var cp = nircmd.spawn('screensaver');
+test('screensaver using spawn', t => {
+	const cp = nircmd.spawn('screensaver');
 
 	cp.on('exit', function (code) {
-		t.assert(code === 0);
+		t.true(code === 0);
+		t.end();
```

Comment:
Doesn't matter that much, but feels like a good practice to define planned assertions.